10-habits/progress/internal/server/getstatus.go

```diff
@@ -18,6 +19,11 @@ import (
 func (s *Server) GetHabitStatus(ctx context.Context, request *api.GetHabitStatusRequest) (*api.GetHabitStatusResponse, error) {
 	log.Printf("GetHabitStatus request received: %s", request)
 
+	err := validateGetHabitRequest(request)
```

Comment:
There was a long discussion about where things must be validated. Let's have it live, just to make sure we have the same strategy.